.travis.yml

```diff
@@ -83,6 +83,7 @@ script:
 after_success:
   - if [ "$COVERAGE" == true ]; then mv cover.out coverage.txt ; else echo 'skipping coverage'; fi
   - if [ "$COVERAGE" == true ]; then bash <(curl -s https://codecov.io/bash) ; else echo 'skipping coverage'; fi
+  - if [ "$COVERAGE" == true ]; then bash cd cmd/opentelemetry/ && <(curl -s https://codecov.io/bash) ; else echo 'skipping coverage'; fi
```

Comment:
it would be good to find a way to use retries here, recently codecov.io has been slow/unstable and these submits fail or timeout.

Reply:
I have added `travis_retry`. IIRC @jpkrohling added it to other jobs.